npg_ranger/README.rst

```diff
@@ -35,12 +38,11 @@ Pending
 
 #. Further work is needed to change references
 
-   * Format our references so they are compatible with browser (bigbed)
-   * Which references?
-   * Where to put them?
-   * Allow user to select reference?
    * Automatic reference from study?
```

Comment:
Strictly speaking, references are per sample. The reference can be inferred from the header. However, this means that a reference track can only be added after a bam file is received, which might be chicken and egg problem. Therefore, we shoudl give a choice of start up file for references we have. As an enchancement, we can validate the reference in the header against a chosen one.